Sources/PointFree/Account/Change.swift

```diff
@@ -56,12 +56,20 @@ private func subscriptionChange(_ conn: Conn<StatusLineOpen, (Stripe.Subscriptio
 
     let (currentSubscription, _, _, newPricing) = conn.data
 
-    let newPrice = defaultPricing(for: newPricing.lane, billing: newPricing.billing) * 100
-    let currentPrice = currentSubscription.plan.amount.rawValue
+    let newPrice = (defaultPricing(for: newPricing.lane, billing: newPricing.billing) * 100) * newPricing.quantity
+    let currentPrice = currentSubscription.plan.amount.rawValue * currentSubscription.quantity
 
     let prorate = newPrice > currentPrice
 
     return AppEnvironment.current.stripe.updateSubscription(currentSubscription, newPricing.plan, newPricing.quantity, prorate)
+      .flatMap { sub -> EitherIO<Error, Stripe.Subscription> in
+        if prorate {
+          parallel(AppEnvironment.current.stripe.invoiceCustomer(sub.customer).run)
```

Comment:
I'm pretty sure I was seein automatic proration when omitting the flag entirely, but in testing was seeing that it wouldn't invoice when explicit.

Reply:
So this is a fire and forget invoice? What happens if it fails? Does that matter?

Reply:
Ah i can do an email